modules/packages/maven/metadata.go

```diff
@@ -109,3 +131,20 @@ func ParsePackageMetaData(r io.Reader) (*Metadata, error) {
 		Dependencies: dependencies,
 	}, nil
 }
+
+// ParseMavenMetadata parses the Maven metadata XML to extract the build number.
+func ParseMavenMetaData(r io.Reader) (string, error) {
+	var metadata MavenMetadata
+
+	dec := xml.NewDecoder(r)
+	dec.CharsetReader = charset.NewReaderLabel // Assuming charset.NewReaderLabel is a function you've set up to handle character encoding.
```

Comment:
Erm...
I mean, it should probably work if I'm reading the documentation correctly, but the comment seems really out of place
```suggestion
	dec.CharsetReader = charset.NewReaderLabel
```